Saudox/resources/views/profissional/anamnese/psicopedagogia/criar.blade.php

```diff
@@ -1,5 +1,1161 @@
+                            <input type="text" name="audicao_e_visao" value="{{ old2($anamnese, "audicao_e_visao") }}" />
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+                        </div>
+                        <div class="pag_form">
+                            <br>
+                            <br>
+                            <hr class="hr_form">
+                            <h1>Sono</h1>
+                            <br>
+
+                            <h3>Sono tranquilo ou agitado? Se é agitado, quando acontece? Com que frequência</h3>
+                            <input type="text" name="sono_tranquilo_se_for_agitado_quando_e_qual_frequencia" value="{{ old2($anamnese, "sono_tranquilo_se_for_agitado_quando_e_qual_frequencia") }}" />
+
+                            <h3>Ranger de dentes</h3>
+                            <input type="radio" name="range_dentes" value="1" {{ old_checked($anamnese, "range_dentes", "1") }} /><label>Sim</label><br>
+                            <input type="radio" name="range_dentes" value="0" {{ old_checked($anamnese, "range_dentes", "0") }} /><label>Não</label><br>
+
+                            <h3>Terror noturno</h3>
+                            <input type="radio" name="terror_noturno" value="1" {{ old_checked($anamnese, "terror_noturno", "1") }} /><label>Sim</label><br>
+                            <input type="radio" name="terror_noturno" value="0" {{ old_checked($anamnese, "terror_noturno", "0") }} /><label>Não</label><br>
+
+                            <h3>Sonambulismo</h3>
+                            <input type="radio" name="sonambulistmo" value="1" {{ old_checked($anamnese, "sonambulistmo", "1") }} /><label>Sim</label><br>
+                            <input type="radio" name="sonambulistmo" value="0" {{ old_checked($anamnese, "sonambulistmo", "0") }} /><label>Não</label><br>
+
+                            <h3>Enurese</h3>
+                            <input type="radio" name="enurese" value="1" {{ old_checked($anamnese, "enurese", "1") }} /><label>Sim</label><br>
+                            <input type="radio" name="enurese" value="0" {{ old_checked($anamnese, "enurese", "0") }} /><label>Não</label><br>
+
+                            <h3>Fala dormindo</h3>
+                            <input type="radio" name="fala_durante_sono" value="1" {{ old_checked($anamnese, "fala_durante_sono", "1") }} /><label>Sim</label><br>
+                            <input type="radio" name="fala_durante_sono" value="0" {{ old_checked($anamnese, "fala_durante_sono", "0") }} /><label>Não</label><br>
+
+                            <h3>Sono tranquilo ou agitado? Se é agitado, quando acontece? Com que frequência</h3>
+                            <textarea class="textareas_form" name="dorme_so_se_nao_com_quem_dorme" rows="4" cols="50" style="">{{ old2($anamnese, "dorme_so_se_nao_com_quem_dorme") }}</textarea>
+
+                            <h3>Até quando dormiu com os pais?</h3>
```

Comment:
Após o campo de textarea o próximo título está ficando muito junto (mesmo caso pra todos), se isso for difícil de resolver, ignorar.
![image](https://user-images.githubusercontent.com/25617786/95790268-6c44e380-0cb5-11eb-9c9b-3040a62e4425.png)

__Acho que com css fica safe de resolver__